chia/_tests/blockchain/test_blockchain.py

```diff
@@ -1845,7 +1845,7 @@ async def test_conditions(self, empty_blockchain, opcode, with_garbage, bt):
         )
         # Ignore errors from pre-validation, we are testing block_body_validation
         repl_preval_results = replace(pre_validation_results[0], error=None, required_iters=uint64(1))
-        code, err, state_change = await b.add_block(blocks[-1], repl_preval_results)
+        code, err, state_change = await b.add_block(blocks[-1], repl_preval_results, None)
```

Comment:
Idea: you could make `add_block` default to `None` and then you don't need to change this line.